docs/general/xs/functions.md

```diff
@@ -1063,16 +1063,16 @@ Prototype: `#!cpp void xsTaskAmount(int taskFieldId, float value)`
 Parameters:
 
 1.  `#!cpp int taskFieldId`: Specifies which property of the task to change
- - 0: Work Value 1
- - 1: Work Value 2
- - 2: Work Range
- - 3: Work Flag 
- - 4: Search Wait Time
- - 5: Unused Flag (it is not actually unused, just what it's called in A.G.E.)
- - 6: Target Diplomacy
+    - 0: Work Value 1 (For aura tasks its stores the amount of attribute that will be gained by the effected units)
```

Comment:
adding 4 spaces might start a new inner instead of continuing the current one

Reply:
✅ That works! _(ignore incorrect code blocks in img, GitHub doesn't support the syntax)_

---

![image](https://github.com/user-attachments/assets/a279d548-2f4c-49b5-8d1f-acb19493ca0b)